### DISCUSSION TITLE
No Laser Light is coming

### DISCUSSION THREAD

**Discussion by LaserBaba** [arkypita/LaserGRBL#2166] [General]
Hi! The Laser was engraving then it continued without actually engraving. The machine is moving but no laser light is appearing. It said- engraving complete but as fact there is no work done.

**Comment by Tribo-2dot0** [comment]
Verifique a conexão do cabo do laser e os fios que o compõem, pode ter havido mal contato.

**Reply by LaserBaba** [reply]
All cables checked. all connected properly, there is no any cut , all fixed properly. It was engraving well, but in the middle of the work the laser light started to decrease and disssapeared

**Reply by Tribo-2dot0** [reply]
Talvez o diodo tenha morrido. Podes ligar o módulo diretamente à fonte de energia (Positivo e Negativo), se normal o laser deve ligar em potência máxima.
